patches/add-feature-flag-based-on-homeserver/matrix-react-sdk+3.71.1.patch

```diff
@@ -0,0 +1,42 @@
+diff --git a/node_modules/matrix-react-sdk/src/IConfigOptions.ts b/node_modules/matrix-react-sdk/src/IConfigOptions.ts
+index b2e44f2..28ff388 100644
+--- a/node_modules/matrix-react-sdk/src/IConfigOptions.ts
++++ b/node_modules/matrix-react-sdk/src/IConfigOptions.ts
+@@ -192,3 +192,12 @@ export interface ISsoRedirectOptions {
+     immediate?: boolean;
+     on_welcome_page?: boolean;
+ }
++
++
++//:tchap: add tchap specific fields
++export interface IConfigOptions {
```

Comment:
I'm confused about this. There is already a IConfigOptions declared in the file, is it overridden ? 

What is this used for ? Validating the fields ? Can you add a comment to explain ?